.github/workflows/helm-test-chart.yml

```diff
@@ -0,0 +1,43 @@
+name: Test Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  test-helm-chart:
+    name: Test Helm Chart
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+
+      - name: Lint the Helm chart
+        run: |
+          helm lint ./chart
+
+      - name: Create Kind cluster
+        uses: helm/kind-action@v1
+
+      - name: Install Helm chart in Kind cluster (Dry Run)
+        run: |
+          helm dependency update ./chart
```

Comment:
why do we run update here?

Reply:
It needs to locally pull down the dependent charts for testing (like meilisearch, rabbit, etc)